int16 roll
int16 pitch
int16 yaw
float32 depth
float32 pos_x
float32 pox_y
float32 pos_z